<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commonFilter</class>
 <widget class="QDialog" name="commonFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Общие фильтры</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>С:</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeFrom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>До:</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeTo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Событие/описание:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editEvent">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFilter">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Фильтр</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Таг/коментарий:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editComment">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
